components/HeaderItems.tsx

```diff
@@ -0,0 +1,57 @@
+import DropdownAux from './DropdownAux';
+
+import { useState } from "react";
+
+//TODO: how to instance this items type
+const HeaderItems = ({ items, mobile }: { items: any, mobile?: Boolean }) => {
+  const [dropdown, setDropdown] = useState(false);

```

Comment:
"dropdown" is not a good variable name in that case since don't tell anything about what the state is about. Maybe "isDropdownActivate"?